Makefile

```diff
@@ -68,18 +68,28 @@ start:
 		--webhook-config-server-port=$(WEBHOOK_CONFIG_PORT) \
 		--webhook-config-mode=$(WEBHOOK_CONFIG_MODE) \
 		--gardener-version="v1.39.0" \
+		--heartbeat-namespace=$(EXTENSION_NAMESPACE) \
+		--heartbeat-renew-interval-seconds=30 \
+		--gardenlet-manages-mcm=true \
+		--webhook-config-service-port=443 \
+		--metrics-bind-address=:8080 \
+		--health-bind-address=:8081 \
 		$(WEBHOOK_PARAM)
 
 .PHONY: start-admission
 start-admission:
-	@LEADER_ELECTION_NAMESPACE=garden GO111MODULE=on go run \
+	@LEADER_ELECTION_NAMESPACE=$(EXTENSION_NAMESPACE) go run \
 		-ldflags $(LD_FLAGS) \
 		./cmd/$(EXTENSION_PREFIX)-$(ADMISSION_NAME) \
 		--webhook-config-server-host=0.0.0.0 \
 		--webhook-config-server-port=$(WEBHOOK_CONFIG_PORT) \
 		--webhook-config-mode=$(WEBHOOK_CONFIG_MODE) \
         $(WEBHOOK_PARAM)
 
+.PHONY: hook-me
+hook-me:
+	bash $(GARDENER_HACK_DIR)/hook-me.sh $(EXTENSION_NAMESPACE) $$(kubectl get namespace | grep $(NAME) | head -n1| cut -d" " -f1) $(WEBHOOK_CONFIG_PORT)
```

Comment:
No it's inconsistent to https://github.com/gardener/gardener-extension-provider-openstack/blob/4531784c98d2e6339bc94f296f56484bb83bfa37/Makefile#L90. Why do we do it differently here?

Reply:
This has a bit more smart to find the proper namespace of the extension in a seed which usually has a suffix. It is not particularly smart but it does a better job at targeting the correct ns.